doc/user_guide/configuration.rst

```diff
@@ -181,3 +181,45 @@ iteration-spaces        This contains definitions of additional iteration spaces
                         in the :ref:`gocean1.0-configuration` section of the
                         GOcean1.0 chapter.
 ======================= =======================================================
+
```

Comment:
Not your fault but earlier in this document we say "and a an optional". Please could you fix.

Reply:
Done